index.js

```diff
@@ -105,6 +105,8 @@ var genericValidationRules = {
 
 function validateGenericEvent (event) {
   assert(type(event) === 'object', 'You must pass a message object.')
+  assert((JSON.stringify(event)).length < 32 << 10, 'Your message must be < 32kb.')
```

Comment:
I'd like it if we moved `32 << 10` into a constant so this line reads a bit nicer. Something like:

```
const MAX_SIZE = 32 << 10

module.exports = looselyValidateEvent

// ...

function validateGenericEvent (event) {
  // ...
  assert(JSON.stringify(event).length < MAX_SIZE, 'Your message must be < 32kb.')
  // ...
}
```

Reply:
Yeah I thought about pulling it out as as a constant. I put it here so that in case this number changes in the future, it's easier to remember we update the error message too. May not be worth worrying about though.

Reply:
Moved to a constant!